scripts/start.js

```diff
@@ -0,0 +1,19 @@
+import * as dotenv from 'dotenv';
+import Webex from 'webex';
+
+import WebexSDKAdapter from '../src/WebexSDKAdapter';
+
+// Allows convict module to read the corresponding configuration
+dotenv.config({path: `${__dirname}/../.env.development`});
+
+if (process.env.WEBEX_ACCESS_TOKEN) {
+  const webex = new Webex({
+    credentials: process.env.WEBEX_ACCESS_TOKEN,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const webexSDKAdapter = new WebexSDKAdapter(webex);
+} else {
+  // eslint-disable-next-line no-console
+  console.error('Please set a valid WEBEX_ACCESS_TOKEN environment in `env.development` file at the root level.');
```

Comment:
Can we `return process.exit(1);` here so the node script knows it failed?

Reply:
then I could get rid of the `else` condition. just have a quick `if` statement at top.